graphqlapi/src/test/java/io/stargate/graphql/schema/GraphQlTestBase.java

```diff
@@ -97,6 +97,9 @@ public void setupEnvironment() {
         GraphQL.newGraphQL(graphQlSchema)
             // Use parallel execution strategy for mutations (serial is default)
             .mutationExecutionStrategy(new AsyncExecutionStrategy())
+            // 24-Jun-2022, tatu: As per [stargate#1279] prevent graphql-java from adding
```

Comment:
@olim7t I hope there are no other side effects to be concerned about here? If there are, can just create an alternative test-exception-handler for the same purpose. But for now assuming production one works well.